drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-client/src/main/java/org/drools/workbench/screens/testscenario/client/reporting/TestRunnerReportingViewImpl.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+
+<!--
+  ~ Copyright 2018 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div data-field="resultPanel" class="container-fluid">
+    <div class="row">
+        <span data-field="resultTitle"></span>
+    </div>
+
+    <div class="row" style="text-align: center;">
+        <div class="col-md-12">
+            <span data-field="resultStats"></span>
```

Comment:
You're declaring `resultStats` as span in the template, but your view has `HTMLDivElement resultStats`. This will lead to errai warnings in the browser console. I'd recommend using div in the template instead.